src/Message/FetchSubscriptionRequest.php

```diff
@@ -131,6 +133,8 @@ public function getData()
             $data['vid'] = $this->getSubscriptionReference();
         }
 
+        $data['startTime'] = $startTime;
+        $data['endTime'] = $endTime;
```

Comment:
the changes in this file are unnecessary... they would add startTime and endTime parameters to the FetchSubscriptionRequest, but that API call doesn't use them.